Pouvoir mettre un lien direct vers une session.

batmat opened the pull request:
Premier jet d'un ajout possible de lien direct vers une session. N'hésitez pas à faire des retours si l'idée vous paraît saugrenue/inappropriée.

batmat committed:
Pouvoir mettre un lien direct vers une session.

fsarradin commented on baggers.html:
```diff
@@ -196,7 +196,7 @@
                                             <ul class="icons">
                                                 {{#sessions}}
-                                                    <li>
+                                                    <li{{#anchor}}id="{{anchor}}"{{/anchor}}>
```
Peut-être à préfixer par le baggerID ?

``` html
<li{{#anchor}}id="{{baggerId}}-{{anchor}}"{{/anchor}}>
```

Un truc comme ça...

batmat replied:
Oui, j'y avais pensé et je m'étais dit qu'on pourrait le faire par norme, mais tu as sûrement raison c'est plus sûr pour éviter les collisions d'id.

MichaelBitard commented:
On ne pourrait pas directement créer une anchor à partir du title plutôt que de rajouter une info "technique" dans le json?

batmat commented:
Pourquoi pas, mais ça implique qu'un changement de titre va faire changer
l'éventuel "permalien".

Ou alors, l'intermédiaire : si pas l'info dans le json, alors on calcule un
anchor à partir du title ?

Le 13 juin 2014 10:08, Michaël Bitard notifications@github.com a écrit :

> On ne pourrait pas directement créer une anchor à partir du title plutôt
> que de rajouter une info "technique" dans le json?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/nrichand/BrownBagLunch/pull/228#issuecomment-45985866
> .

## 

Baptiste <Batmat> MATHUS - http://batmat.net
Sauvez un arbre,
Mangez un castor !

MichaelBitard commented:
Bien vu le permalien qui peut changer, du coup je suis d'accord avec ta proposition intermédiaire.

batmat committed:
Préfixage baggerId, pour éviter les collisions

batmat committed:
Génération ancre par défaut si attribut absent.

Le code de gestion des accents est particulièrement énorme,
je l'ai préféré à la solution regexp pour favoriser les perfs.
Si vous préférez l'option lisibilité ici, on peut faire autrement :-).

batmat commented:
Done
